src/components/collections/collection-card.tsx

```
@@ -0,0 +1,144 @@
+                  className="h-8 w-8"
+                  onClick={() => setShowMenu(!showMenu)}
+                >
+                  <MoreHorizontal className="h-4 w-4" />
+                </Button>
+
+                {showMenu && (
+                  <div className="absolute right-0 top-full mt-1 w-48 bg-popover border rounded-lg shadow-lg z-10 py-1">
+                    <button
+                      className="w-full px-4 py-2 text-left text-sm hover:bg-accent flex items-center gap-2"
+                      onClick={handleCopyLink}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)